parquet-cli/src/main/java/org/apache/parquet/cli/commands/ShowBloomFilterCommand.java

```diff
@@ -63,14 +67,31 @@ public ShowBloomFilterCommand(Logger console) {
       required = true)
   List<String> testValues;
 
+  @Parameter(
+      names = {"--footer-key"},
+      description = "Footer key for encrypted files (hex string, 16/24/32 bytes)")
+  String footerKeyHex;
+
+  @Parameter(
+      names = {"--column-key"},
+      description = "Column key for encrypted files (hex string, 16/24/32 bytes)")
+  String columnKeyHex;
+
```

Comment:
@ArnavBalyan, is it only bloom filters? Shouldn't we add this at a more common level, so other commands would also benefit from it?